python/examples/langchain-cdp-chatbot/pyproject.toml

```diff
@@ -2,6 +2,7 @@
 name = "chatbot-python"
 version = "0.0.1"
 description = "CDP AgentKit Example Chatbot"
+private = true
```

Comment:
This is a non-standard field, but is a concept borrowed from NPM to denote that this package should not be published